source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,591 @@
+module Fluent
+
+  class OutChangeTrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
```

Comment:
the conf file using this is generated dynamically. I want to make tags and plugin name related, so i can generate it dynamically? What is your exact opposition here? I can change the name to "out.oms.changetracking.file",  but I cannot do "out_oms_changetracking_file" since this is a bigger change for me, which I dont think is worthwhile.